src/main/java/com/github/instagram4j/instagram4j/requests/accounts/AccountsEditStatusRequest.java

```diff
@@ -0,0 +1,55 @@
+package com.github.instagram4j.instagram4j.requests.accounts;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.instagram4j.instagram4j.IGClient;
+import com.github.instagram4j.instagram4j.models.IGPayload;
+import com.github.instagram4j.instagram4j.requests.IGPostRequest;
+import com.github.instagram4j.instagram4j.responses.IGResponse;
+import com.github.instagram4j.instagram4j.responses.accounts.AccountsUserResponse;
+
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.Accessors;
+
+@RequiredArgsConstructor
+public class AccountsEditStatusRequest extends IGPostRequest<IGResponse> {
```

Comment:
Please change the name to StatusSetStatusRequest (I know very strange!)

The naming convention for request is usually the value in path. (I assume the full request endpoint is api/v1/**status/set_status/**).

This should be located under a separate package (called "status"). All requests under "accounts" are all requests falling under "accounts" hierarchy. So this should be under a package called "status" since this falls into "status" hierarchy.

I'm sorry to be very particular about this.